index.html

```diff
@@ -157,7 +157,7 @@ <h2>Data Model</h2>
             <a>verifiable credential</a>.
          </p>
          <p>
-            Implementers may find use in packaging a JSON Schema as a verifiable credential when they wish to
+            Implementers MAY find use in packaging a JSON Schema as a verifiable credential when they wish to
```

Comment:
This normative language is strange, suggest changing it to "might".

Alternatively, you could say: "Implementers MAY package a JSON Schema as a verifiable credential when they..."

Reply:
thanks, updating